packages/manager/modules/octavia-load-balancer/src/octavia-load-balancer.module.js

```diff
@@ -7,12 +7,13 @@ import 'angular-translate';
 import ngOvhUtils from '@ovh-ux/ng-ovh-utils';
 import uiKit from '@ovh-ux/ui-kit';
 
-import { ListLayoutHelper } from '@ovh-ux/manager-ng-layout-helpers';
-
 import dashboard from './dashboard';
 import onboarding from './onboarding';
 import routing from './octavia-load-balancer.routing';
 import create from './create';
+import deleteComponent from './delete';
```

Comment:
```suggestion
import delete from './delete';
```

Reply:
Hello, "delete" is a reserved javascript keyword, that's why I had to use "deleteComponent" instead.